### DISCUSSION TITLE
is input_boolean.xplora_tracker_switch deprecated?

### DISCUSSION THREAD

**Discussion by sebatze** [Ludy87/xplora_watch#103] [Q&A] [answered]
Hi, are the input booleans "input_boolean.xplora_tracker_switch" and "input_boolean.xplora_track_now" deprecated (https://github.com/Ludy87/xplora_watch/issues/15)?

I assume this has been replaced by the "Track Watch" service, but I want to double check to be sure

**Comment by Ludy87** [comment, marked as answer]
hi, yes it is out of date due to the new service.

**Reply by sebatze** [reply]
thanks!
